bin/packages/build.js

```diff
@@ -54,12 +97,22 @@ function createStyleEntryTransform() {
 				return;
 			}
 
-			packages.add( packageName );
-			const entries = await glob(
-				path.resolve( PACKAGES_DIR, packageName, 'src/*.scss' )
-			);
-			entries.forEach( ( entry ) => this.push( entry ) );
-			callback();
+			// Find other stylesheets that need to be rebuilt because
+			// they import the styles that are being transformed
+			const styleEntries = findStyleEntriesThatImportFile( file );
+
+			// Rebuild stylesheets that import the styles being transformed
+			if ( styleEntries.length ) {
+				styleEntries.forEach( ( entry ) => stream.push( entry ) );
+				callback();
+			} else {
+				packages.add( packageName );
+				const entries = await glob(
+					path.resolve( PACKAGES_DIR, packageName, 'src/*.scss' )
+				);
+				entries.forEach( ( entry ) => this.push( entry ) );
```

Comment:
Should this section execute regardless of whether or not there were `styleEntries` found above?  Im still trying to wrap my head around what is happening here, but it seems this is the package we want transformed and should still be doing that regardless of where it is being imported?  Should this be something like:
```
		packages.add( packageName );
		const entries = await glob(
			path.resolve( PACKAGES_DIR, packageName, 'src/*.scss' )
		);
		entries.forEach( ( entry ) => this.push( entry ) );

		// Find other stylesheets that need to be rebuilt because
		// they import the styles that are being transformed
		const styleEntries = findStyleEntriesThatImportFile( file );

		// Rebuild stylesheets that import the styles being transformed
		if ( styleEntries.length ) {
			styleEntries.forEach( ( entry ) => stream.push( entry ) );
		}
		callback();

Reply:
Great callout! 

You're totally right. It's curious that what I wrote still works 🤔 , but I have a good guess as to why. I'll make sure to update the conditional logic to reflect what you wrote.